modules/react-components/src/components/input/advanced-search.tsx

```diff
@@ -307,8 +307,11 @@ export const AdvancedSearch: FunctionComponent<PropsWithChildren<AdvancedSearchP
                 const terms: string[] = internalSearchQuery.split(" ");
 
                 if (terms.length > 2) {
+                    const filterAttributeParts = terms[0].split(".");
+                    const filterAttribute = filterAttributeParts.length > 1 ? filterAttributeParts[0] : terms[0];
+
                     const attributes = filterAttributeOptions.filter((attribute) => {
-                        return attribute.value === terms[0];
+                        return attribute.value === filterAttribute;
```

Comment:
Can you check whether this change affects other places where we have used this reusable component?

Reply:
Resolving as per the offline discussion